src/components/v2/EnableToken/index.tsx

```diff
@@ -0,0 +1,80 @@
+/** @jsxImportSource @emotion/react */
+import React, { useContext } from 'react';
+import BigNumber from 'bignumber.js';
+import Typography from '@mui/material/Typography';
+import { AuthContext } from 'context/AuthContext';
+import { TokenSymbol } from 'types';
+import { formatApy } from 'utilities/common';
+import useApproveToken from 'clients/api/mutations/useApproveToken';
+import { Icon, IconName } from '../Icon';
+import { PrimaryButton } from '../Button';
+import useStyles from './styles';
+
+export interface IEnableTokenProps {
+  symbol: TokenSymbol;
+  isEnabled: boolean;
+  title: string;
+  tokenInfo: { symbol: TokenSymbol; text: string; apy: BigNumber | number }[];
+  approveToken: () => void;
+  vtokenAddress: string;
+  disabled: boolean;
+}
+
+export const EnableTokenUi: React.FC<Omit<IEnableTokenProps, 'vtokenAddress'>> = ({
+  symbol,
+  title,
+  tokenInfo,
+  isEnabled,
+  children,
+  approveToken,
+  disabled,
+}) => {
+  const styles = useStyles();
+  if (isEnabled) {
+    return <>{children}</>;
+  }
+  return (
+    <div css={styles.container}>
+      <Icon name={symbol as IconName} css={styles.mainLogo} />
+      <Typography component="h3" variant="h3" css={styles.mainText}>
+        {title}
+      </Typography>
+      <hr />
```

Comment:
We have a separate component for that: `Delimiter`. It might not be necessary though